application/values-test.yaml

```diff
@@ -628,6 +628,45 @@ externalSecret:
       labels:
         key: value
 
+##########################################################
+# Network Policy
+##########################################################
+networkPolicy: 
+  enabled: false
+  # Additional labels
+  additionalLabels:
+    key: value
+
+  # Additional annotations
+  annotations:
+    # key: value
+  
+  # Ingress rules
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 172.17.0.0/16
+        except:
+        - 172.17.1.0/24
+    - namespaceSelector:
+        matchLabels:
+          project: myproject
+    - podSelector:
```

Comment:
I dont think so we should be adding it over here. it should have been picked by application itself